### DISCUSSION TITLE
Progress Episodes direct to play, not to remaining episodes list

### DISCUSSION THREAD

**Discussion by EvanderMegaton** [umbrellaplug/umbrellaplug.github.io#414] [General]
Hello everyone.
Is it possible in Umbrela, after creating a widget from the Progress Episodes list, to immediately start the source search without having to select the episode again on the next screen? I'm trying to find a setting that would enable it, but I can't.

Thank you!

**Comment by adamosborne83** [comment]
Autoplay?

**Comment by umbrellaplug** [comment]
You need to enable "Direct Progress Scaping". It's in the addon settings under Settings > Trakt > Direct Progress Scraping

**Reply by EvanderMegaton** [reply]
excellent! that's it. I just didn't know where to look for that option, I didn't assume it was related to the Trakt part. :)
